docs/content/guide/migration.ngdoc

```diff
@@ -276,15 +276,15 @@ statement.
 **Due to [6ccbfa](https://github.com/angular/angular.js/commit/6ccbfa65d60a3dc396d0cf6da21b993ad74653fd)**,
 the `xlink:href` security context for SVG's `a` and `image` elements has been lowered.
 
-In the unlikely case that an app relied on `RESOURCE_URL` whitelisting for the
+In the unlikely case that an app relied on `RESOURCE_URL` trusted list for the
 purpose of binding to the `xlink:href` property of SVG's `<a>` or `<image>`
 elements and if the values do not pass the regular URL sanitization, they will
 break.
 
 To fix this you need to ensure that the values used for binding to the affected
-`xlink:href` contexts are considered safe URLs, e.g. by whitelisting them in
-`$compileProvider`'s `aHrefSanitizationWhitelist` (for `<a>` elements) or
-`imgSrcSanitizationWhitelist` (for `<image>` elements).
+`xlink:href` contexts are considered safe URLs, e.g. by trusting them in
+`$compileProvider`'s `aHrefSanitizationTrustedUri` (for `<a>` elements) or
+`imgSrcSanitizationTrustedUri` (for `<image>` elements).
```

Comment:
FWIW, I find it confusing to update the API names in the migration guide.

Each section of the guide has instructions on migrating to a specific version. For the version discussed here, the new API names do not exist, so the user will not be able to find them.

Maybe we could use the old names (🤷‍♂️) and add in parenthesis next to them something like "called <NEW_NAME> in latest versions".

Reply:
Done.